playground/frontend/test/pages/playground/states/example_selector_state_test.mocks.dart

```diff
@@ -1,7 +1,8 @@
-// Mocks generated by Mockito 5.2.0 from annotations
+// Mocks generated by Mockito 5.3.2 from annotations
```

Comment:
If mockito is upgraded, this should also be in pubspec.lock, otherwise someone will overwrite this with the old version and can face potential merge conflicts.

Reply:
Hm, pubspec.lock hasn't updated at my repo